test/unit/lib/plugins/aws/remove/lib/bucket.test.js

```diff
@@ -80,6 +80,66 @@ describe('emptyS3Bucket', () => {
     });
   });
 
+  describe('#listObjectVersions()', () => {
+    it('should resolve if no object versions are present', () => {
+      const listObjectVersionsStub = sinon.stub(awsRemove.provider, 'request').resolves();
```

Comment:
All new tests should be configured with `runServerless `util.

It's documented here: https://github.com/serverless/test/blob/master/docs/run-serverless.md

Also search the code for many examples on how it's used.

I believe in this given case we need to stub AWS SDK with `awsRequestStubMap` option. And via stubs we should confirm that expected objest from S3 bucket were attempted to be removed